src/h5web/vis-packs/nexus/containers/NxSpectrumContainer.tsx

```diff
@@ -34,6 +35,12 @@ function NxSpectrumContainer(props: VisContainerProps): ReactElement {
 
   const [dimMapping, setDimMapping] = useDimMappingState(signalDims, 1);
 
+  const { valuesStore } = useContext(ProviderContext);
+  valuesStore.prefetch({ path: signalDataset.path });
+  if (errorsDataset) {
+    valuesStore.prefetch({ path: errorsDataset.path });
+  }
```

Comment:
As discussed, I had to remove the prefetching of the signal from `useNxData`, so I decided to do the same with errors for consistency and move the prefetching here, in `NxSpectrumContainer`.